pkg/analyzer/analyzers/mailchimp/mailchimp.go

```diff
@@ -26,11 +28,75 @@ type Analyzer struct {
 func (Analyzer) Type() analyzerpb.AnalyzerType { return analyzerpb.AnalyzerType_Mailchimp }
 
 func (a Analyzer) Analyze(_ context.Context, credInfo map[string]string) (*analyzers.AnalyzerResult, error) {
-	_, err := AnalyzePermissions(a.Cfg, credInfo["key"])
+	key, ok := credInfo["key"]
+	if !ok {
+		return nil, errors.New("key not found in credentialInfo")
+	}
+
+	info, err := AnalyzePermissions(a.Cfg, key)
 	if err != nil {
 		return nil, err
 	}
-	return nil, fmt.Errorf("not implemented")
+	return secretInfoToAnalyzerResult(info), nil
+}
+
+func secretInfoToAnalyzerResult(info *SecretInfo) *analyzers.AnalyzerResult {
+	if info == nil {
+		return nil
+	}
+	result := analyzers.AnalyzerResult{
+		AnalyzerType:       analyzerpb.AnalyzerType_Mailchimp,
+		Bindings:           make([]analyzers.Binding, 0, len(StringToPermission)),
+		UnboundedResources: make([]analyzers.Resource, len(info.Domains.Domains)),
```

Comment:
**Nit:** Just for consistency I think it would be better for these to be allocated the same way (i.e. both allocate `cap` or both allocate `len`).